### PR TITLE
added some db

### PR DESCRIPTION
**Pull request by Sanjeewaniaryal**
please reviewthis file

### CODE REVIEW

**Review by deepakbaral — COMMENTED** [review]
This is good.

**Review by ujjwal1 — COMMENTED** [review]
Reviewed.

File: sanjeewani/sanjeewani.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Page Title</title>
```
Comment (itssabhishekk) [inline sanjeewani/sanjeewani.html:4]: Would be better if the title was your name

**Review by itssabhishekk — APPROVED** [review]
Seems fine. Approved it.

**Pull request merged**